### PR TITLE
Release 1.13.1

### PR DESCRIPTION
**Pull request by Jkovarik**
See changelog for fixes backported. 

Everything in ./bamboo/ is CI related. 

Fixes: 

Added a priority port to fix the CMR env from CUMULUS-1345
Updated googleApi package to resolve audit failure
Updated elasticsearch module to allow us to update lodash to resolve audit failure.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [v.13.1] - 2019-07-22
```
Comment (laurenfrederick) [inline CHANGELOG.md]: At the bottom of this doc, add a link for the comparison between the versions.

**Comment by laurenfrederick** [comment]
Oh and the docs should be built too for this version.

File: CHANGELOG.md (hunk 1 above)
Comment (Jkovarik) [reply CHANGELOG.md]: 👍  I hadn't run through the deploy doc yet. 

150235450

**Comment by Jkovarik** [comment]
Closing as 1.13.1 has been released.

**Pull request closed without merging**